scripts/npm/with/storage.js

```diff
@@ -0,0 +1,27 @@
+const FS = require('fs');
+
+class Storage {
```

Comment:
I do not understand the purpose of the storage class. Is it so that you save the parameters in a file so you don't have to write then every time?

Reply:
I didn't find it makes sense that saving the given parameters is parser's (arguments parser's) job.
So, Storage class is just a small abstraction.